packages/frontend/src/scripts/forced-actions/spotWithdrawalForm.ts

```diff
@@ -0,0 +1,14 @@
+import { SpotWithdrawalFormId } from '../../view/pages/forced-actions/NewSpotForcedWithdrawalPage'
+
+export function initSpotWithdrawalForm() {
+  const form = document.getElementById(SpotWithdrawalFormId)
+
+  if (!form) {
+    return
+  }
+
+  form.addEventListener('submit', (e) => {
+    e.preventDefault()
+    alert('Spot withdrawal submitted')
```

Comment:
Do we want to leave this as a placeholder here?

Reply:
yup